dojo-dungeon/src/dojo/rooms/Room.java

```diff
@@ -140,4 +140,13 @@ public void closeDoors() {
   public Tile getStartTile() {
     return levelRoom.level().startTile();
   }
+
+  /** Decreases the heros health at a wrong try (minus 25% (ceiling) of maximal health points). */
```

Comment:
Wie kommst Du jetzt auf 25%? Ich meine mich zu erinnern, dass bei einer falschen Antwort 50% abgezogen werden sollten?

Ich würde die Prozente aber unbedingt als Parameter in die Methode reingeben. Dann musst Du die auch nicht überschreiben.

Reply:
Ich fand 25 % (nicht relativ betrachtet) sinnvoller als minus 50 %, sonst hätte man doch nur zwei oder drei Versuche?

Ich ändere das noch, sodass der Prozentanteil vom Maximum übergeben werden kann. Danke dir erst mal fürs Review.

Reply:
naja, die 50% waren zuletzt besprochen. und ja, es gibt damit nur einen fehlversuch... aber das finde ich bei zwei fragen auch sinnvoll.

Reply:
> naja, die 50% waren zuletzt besprochen. und ja, es gibt damit nur einen fehlversuch... aber das finde ich bei zwei fragen auch sinnvoll.

Also, 50 % bezogen auf die max. points? Das wären nur zwei Versuche, ja. 

Man könnte das auch abhängig von der Anzahl der Fragen und der benötigten richtigen Antworten machen. Bei mehr als einer erforderlichen richtigen Antwort wäre nur ein Fehlversuch fast sehr schwer. Hm, also wie siehst du das, bei -50% bleiben, aber via Parameter/Konstante änderbar? ... Oder eine Formel dafür aufstellen?

Reply:
konstante für jetzt. später ... wer weiß :)

Reply:
Jetzt sollte es passen :D